packages/orbit-components/src/InputGroup/InputGroup.stories.tsx

```diff
@@ -78,6 +78,7 @@ export const DateOfBirth: Story = {
     return (
```

Comment:
One weird thing if I use VoiceOver, let's say I select February, it would announce: "February, January"

So it might be an issue with the Select component itself and we will deal with it subsequently?

Reply:
I checked and this is an issue in Firefox, however it's correctly announced in Safari. I'll create a task for Select component to take a look at this.

Reply:
Thank you Domi!